src/Searchable/ModelsResolver.php

```diff
@@ -54,7 +54,7 @@ public function from(Builder $builder, $searchable, array $ids): Collection
 
             if (in_array(Searchable::class, class_uses_recursive($model), true)) {
                 if (! empty($models = $model->getScoutModelsByIds($builder, $modelKeys))) {
-                    $instances = $instances->merge($models);
+                    $instances = $instances->merge($models->load($searchable->getRelations($modelClass)));
```

Comment:
What happens if the relations are an empty array?

Reply:
No errors and nothing will get loaded